yat-lib-example/src/main/java/yat/yat_lib_example/MainActivity.kt

```diff
@@ -127,6 +84,92 @@ internal class MainActivity : AppCompatActivity(), YatIntegration.Delegate {
         displayErrorDialog(errorMessage)
     }
 
+    private fun testIncomingTx() {
+        YatIntegration.processDeepLink(
+            context = this,
+            deepLink = Uri.parse("tari://yat.fyi?action=connect&eid=%F0%9F%A5%9A%F0%9F%91%8D%E2%9D%97%F0%9F%A7%A0%F0%9F%8D%95%F0%9F%A7%B2&refresh_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJlbW9qaS1pZC10b2tlbi1pc3N1ZXIiLCJzdWIiOiJhZWIzMzYxOS0wYmYyLTQwZTUtYWJjYS02MzI3M2Q2ZDgzMTEiLCJleHAiOjE3MjU1MzE2NDksImlhdCI6MTcyNDkyNjg0OSwic2NvcGVzIjpbInRva2VuOnJlZnJlc2giXSwiYWN0aXZlMmZhIjowLCJodHRwczovL2hhc3VyYS5pby9qd3QvY2xhaW1zIjp7IngtaGFzdXJhLWRlZmF1bHQtcm9sZSI6InVzZXIiLCJ4LWhhc3VyYS1hbGxvd2VkLXJvbGVzIjpbInVzZXIiLCJhZG1pbiJdLCJ4LWhhc3VyYS11c2VyLWlkIjoiYWViMzM2MTktMGJmMi00MGU1LWFiY2EtNjMyNzNkNmQ4MzExIn19.uek-qF_GVQ_0305k8Wj6bVHXl6dZ1mgWEE0AMGijDLM")
+        )
+    }
+
+    private fun testOutcomingTx() {
+        val data = YatLibOutcomingTransactionData(
+            amount = 10.2,
+            currency = "ETH",
+            yat = "\uD83D\uDE02\uD83D\uDE07\uD83D\uDE43\uD83D\uDE0D\uD83E\uDD16",
+        )
+
+        YatLibOutcomingTransactionActivity.start(
+            context = this,
+            outcomingTransactionData = data,
+            type = OutcomingTransactionExampleActivity::class.java,
+        )
+    }
+
+    private fun testGetYat() {
+        val yatRecords = listOf(
+//        YatRecord(
+//            type = YatRecordType.ADA_ADDRESS,
+//            data = "DdzFFzCqrhsgwQmeWNBTsG8VjYunBLK9GNR93GSLTGj1FeMm8kFoby2cTHxEHBEraHQXmgTtFGz7fThjDRNNvwzcaw6fQdkYySBneRas",
+//        ),
+//        YatRecord(
+//            type = YatRecordType.DOT_ADDRESS,
+//            data = "GC8fuEZG4E5epGf5KGXtcDfvrc6HXE7GJ5YnbiqSpqdQYLg",
+//        ),
+//        YatRecord(
+//            type = YatRecordType.BTC_ADDRESS,
+//            data = "1NDyJtNTjmwk5xPNhjgAMu4HDHigtobu1s",
+//        ),
+//        YatRecord(
+//            type = YatRecordType.ETH_ADDRESS,
+//            data = "108dEFa0272dC118EF03a7993e4fC7A8AcF3a3d1",
+//        ),
+//        YatRecord(
+//            type = YatRecordType.XTR_PUBLICKEY,
+//            data = "d2e4db6dac593a9af36987a35676838ede4f69684ba433baeed68bce048e111b",
+//        ),
+//        YatRecord(
+//            type = YatRecordType.XMR_STANDARD_ADDRESS,
+//            data = "4AdUndXHHZ6cfufTMvppY6JwXNouMBzSkbLYfpAV5Usx3skxNgYeYTRj5UzqtReoS44qo9mtmXCqY45DJ852K5Jv2684Rge",
+//        ),
```

Comment:
Commented code.

Reply:
@TruszczynskiA It's a sample project, there are a lot of commented examples. I'll add a comment